app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -207,6 +212,35 @@ private void setAchievements() {
                 ));
     }
 
+    /**
+     * To call the API to get reverts count in form of JSONObject
+     *
+     */
+    private void setRevertCount(){
+        compositeDisposable.add(mediaWikiApi
+                .getRevertCount(sessionManager.getCurrentAccount().name)
```

Comment:
Can `sessionManager.getCurrentAccount()` ever return a null? If yes, add a null check.

Reply:
It won't return a null till logging in mandatory :)

Reply:
Check the java docs for this method. :)

Reply:
@maskaravivek this kind of check?  
Optional.ofNullable(sessionManager.getCurrentAccount().name.orElse("")
It requires min API 24 where as min API of Commons app is 15. Any idea how to go about it ?

Reply:
A simple `null` check like this should be enough. 

```
Account currentAccount = sessionManager.getCurrentAccount();
if(currentAccount == null) {
// do something
}
```

Recently we experienced a few crashes on play store and we suspect that `getCurrentAccount` is returning null in some cases even for logged in users. 

Refer to this PR for more context: 
https://github.com/commons-app/apps-android-commons/pull/1684/files